### ISSUE TITLE
Allow viewing star rating of map inside of the server browser

### ISSUE THREAD

**Issue by art0007i** [ddnet/ddnet#6632]
I think it would be cool to see star rating of maps in the server browser.
This could be done by either making the server motd show up somewhere or just show the stars next to the map name.
Or really any other way to display this information within the client before joining a server.
I've drawn a mockup of what I would expect this feature to look like:

![possible implementations mockup](https://github.com/ddnet/ddnet/assets/19620451/852f5ecd-f4bd-406e-92f7-4851f9a4ee0a)

**Comment by def-** [comment]
I could send that info in the serverinfo json for official ddnet server. Anyone interested in coding the UI part? And figuring out how to display it nicely?

**Comment by fokkonaut** [comment]
Maybe hovering over the map name should show the stars.

**Comment by gerdoe-jr** [comment]
Yeah, MOTD panel would be good. Also it can be collapsible, so, if player don't want to see anything they can close this panel.

**Comment by Robyt3** [comment]
I can code the UI part if you add the info to the server info json.

**Comment by def-** [comment]
I just realized that this is a bit awkward to implement. In https://info.ddnet.org/info?name=deen we don't have an association of map to server. And if we'd want to change the https://master1.ddnet.org/ddnet/15/servers.json format for this, it would be a larger change and make the file size much larger if it contains motd, or even stars, points (and the latter two would be ddnet-specific, not generic).

**Comment by Robyt3** [comment]
What if the information about map difficulty came from https://info.ddnet.org/info for each community? I.e. it would specify for every community a list of maps together with attributes like difficulty, average finish time etc. CC #7994.

**Comment by heinrich5991** [comment]
> What if the information about map difficulty came from https://info.ddnet.org/info for each community? I.e. it would specify for every community a list of maps together with attributes like difficulty, average finish time etc.

That seems weird, as it's not really community-specific. Stars or average finish time seem to belong into the server info instead, I think.

MOTD would be too large to add to the server info.

**Comment by gerdoe-jr** [comment]
> MOTD would be too large to add to the server info.

Hm, this could be implemented by another request (abusable?), querying MOTD information separately. Vanilla client will not support it anyways.
This way we could display map information right in the server browser, AFAIR, DDNet servers MOTD contains map information by default. Also third-party server hosters will be able to share more information about server right in MOTD. I don't know if people really read MOTD when join unfamiliar server
